packages/sampler/src/MoonstoneEnvironment/MoonstoneEnvironment.js

```diff
@@ -156,7 +156,7 @@ const StorybookDecorator = (story, config) => {
 			className={classnames(classes)}
 			title={`${config.kind} ${config.story}`.trim()}
 			description={config.description}
-			locale={select('locale', locales, Config, getPropFromURL('locale'))}
+			locale={select('locale', locales, Config, getPropFromURL('locale') || window.navigator.language)}
```

Comment:
We may need to guard against the use of `window` in case it doesn't exist. I'm not sure though.